frontend_v2/src/app/services/auth.service.ts

```diff
@@ -69,7 +68,7 @@ export class AuthService {
      */
     logOut() {
       const temp = {isLoggedIn: false};
-      const routePath = '/auth/login';
+      const routePath = '';
```

Comment:
@Kajol-Kumari Why we are redirecting it to landing page after logout?

Reply:
Hey @Sanji515 I have done this change as per the discussion done in [here](https://github.com/Cloud-CV/EvalAI/issues/2646) 
If it seems to be not needed currently, please let me know. I'll remove it.